### ISSUE TITLE
Find packages that provide the wanted package

### ISSUE THREAD

**Issue by mbloms** [trizen/trizen#118]
Similar to #23. When I write `trizen -S java-runtime` it asks me if I want the `jre8-openjdk-8.u162-0.1` package (which provides `java-runtime`) , which is also what pacman does. However if I run `trizen -Sa java-runtime`, it responds with `Package not found: java-runtime` despite there being multiple packages providing `java-runtime` in the aur.

It would be really nice to be given a list of packages that provide `java-runtime`, when searching for it.

**Comment by trizen** [comment]
> It would be really nice to be given a list of packages that provide java-runtime, when searching for it.

I agree, but currently this feature is not supported by AurJson: https://wiki.archlinux.org/index.php/AurJson

**Comment by AladW** [comment]
Related discussion: https://github.com/actionless/pikaur/issues/53
